pkg/cli/environment.go

```diff
@@ -56,6 +59,8 @@ type EnvSettings struct {
 	RepositoryCache string
 	// PluginsDirectory is the path to the plugins directory.
 	PluginsDirectory string
+	// HistoryMax is the name of the env var for setting max history.
+	HistoryMax int
```

Comment:
Observation: I think for consistency with the arg `--history-max`, `HistoryMax` should remain but really `MaxHistory` would read better.

Reply:
I am also hesitant about this, if ok, I can update to "MaxHistory"

😄😄😄

Reply:
I agree with @hickeyma, `MaxHistory` reads better.

Reply:
Deal, modified.  😄